README.md

```diff
@@ -1 +1,23 @@
+
+**HalcyonGrid** or **HalcyonSim** was a fork from the <a href="https://opensimulator.org" rel="nofollow" >OpenSimulator</a> Project in 2010, and is intended to be a reliable simulator. 
+
+## Virtual World where people may walk in a full 3D world.
+
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "viewer", which is used to access/login to these virtual worlds (not a VR headset). The most common one currently is the <a href="https://firestormviewer.org" rel="follow">Firestorm viewer for OpenSim</a>.  *(the "OS" stands for 'OpenSim', and not 'Operating System')* also it is similar to <a href="https://SecondLife.com" rel="nofollow" >Second Life™</a>.  It allows users to "talk" to other people in-world through nearby chat, IM's, group chats and group IM's and sometimes in voice, depending on how the grid is setup and the permissions set on individual regions. Including offline IM's, sending the offline IM's to the user's registered email address. That is as much like a game as it gets. There is also the ability to build objects in-world known as primatives or "prims" in various shapes and sizes to make different types of linked or individual 3D objects. Additionally other 3D items known as "Mesh" (DEA files) can be imported which have been created/edited with a 3D object editor/creator such as the free program titled "<a href="https://store.steampowered.com/app/365670/Blender/" rel="follow" >Blender</a>", or exported from a world as DEA. 
+
+# Different Type of Grid
+
+Halcyon is different and incompatible with both Second Life™ and OpenSim Grids, although we can use the OpenSim viewers. Halcyon has better physics & script engines, more prims, free upload (depending on each individual world's setting), and a more powerful asset database (Not a "Cloud" ... <a href="https://fsfe.org/contribute/spreadtheword.html#nocloud" rel="follow" >*there is no "cloud" just other people's computers*</a>). As a proof, <a href="https://youtube.com/watch?v=_QnNH-xDPyg" rel="nofollow">this tram</a> has no equivalent in any of the SL or OS worlds. 
```

Comment:
I just checked that FSFE link... Why are you attempting to place marketing materials here?

Reply:
Well I guess you could consider that as "marketing materials", though it's true, there is no "cloud"... just other people's computers. And that's the **Free Software Foundation Europe** website. Hence the "FSFE.org". But you're probably right that that's not necessary to be there, and now that you mention it, that may not actually be true for every Halcyon World, that they're not using the "cloud" computing. Sorry, I have removed it in a newer version.

Thank you, Shalom.